rpcs3/Emu/Memory/vm.cpp

```diff
@@ -905,11 +905,11 @@ namespace vm
 
 	static std::shared_ptr<block_t> _find_map(u32 size, u32 align, u64 flags)
 	{
-		for (u32 addr = ::align<u32>(0x20000000, align); addr < 0xC0000000; addr += align)
+		for (u64 addr = ::align<u32>(0x20000000, align); addr < 0xC0000000; addr += align)
```

Comment:
It's not elegant to cast it to u64 here.
``addr <= (0xB0000000 & -align)`` is an option.

Reply:
I put the check after _test_map because checking before it has the same issue as I attempted to fix.

Reply:
`addr && addr < 0xC0000000` is also an option.